### PR TITLE
Update university domain list

### PR DESCRIPTION
**Pull request by Qt-nhannguyen**
QTBSI-147: Update university domain list Vol. 22
QTBSI-227: Update university domain list Vol. 23

### CODE REVIEW

File: world_universities_and_domains.json [hunk 1]
```diff
@@ -126469,5 +126472,773 @@
...
+    "domains": [
+      "stu.nau.edu.cn"
+    ],
+    "country": "China"
+  },
+  {
+    "web_pages": [
+      "https://ju.se/"
+    ],
+    "name": "Jönköping Univerisity",
```
Comment (ollipuu) [inline world_universities_and_domains.json:126672]: Looks like a typo here

File: world_universities_and_domains.json [hunk 2]
```diff
@@ -126469,5 +126472,773 @@
...
+    "domains": [
+      "dtu.edu.vn"
+    ],
+    "country": "Vietnam"
+  },
+  {
+    "web_pages": [
+      "www.kafu.ac.ke"
+    ],
+    "name": "KAIMOSI FRIENDS UNIVERSITY",
```
Comment (ollipuu) [inline world_universities_and_domains.json:126696]: I guess shouldn't be in all caps

File: world_universities_and_domains.json [hunk 3]
```diff
@@ -126469,5 +126472,773 @@
...
+    "domains": [
+      "untels.edu.pe"
+    ],
+    "country": "Peru"
+  },
+  {
+    "web_pages": [
+      "https://www.mondragon.edu/en/home"
+    ],
+    "name": "MONDRAGON UNIBERTSITATEA",
```
Comment (ollipuu) [inline world_universities_and_domains.json:126972]: All caps here as well

File: world_universities_and_domains.json [hunk 4]
```diff
@@ -126469,5 +126472,773 @@
...
+    "web_pages": [
+      "https://www.artez.nl/"
+    ],
+    "name": "Artez University of the Arts",
+    "alpha_two_code": "NL",
+    "state-province": null,
+    "domains": [
+      "student.artez.nl"
+    ],
+    "country": "Netherland"
```
Comment (ollipuu) [inline world_universities_and_domains.json:127086]: Should be _Netherlands_

**Review by ollipuu — CHANGES_REQUESTED** [review]
I don't think it should be our job to proof read these. The letter case differences don't matter that much but wrong country name will surely cause issues in SFDC validation. I didn't go through each single one so there might be something else as well.

**Comment by Qt-nhannguyen** [comment]
Good catches Olli, I have corrected some typos at another university but am still missing those. Thanks a lot!

**Comment by Qt-nhannguyen** [comment]
Another scan was done on the added universities and I could not find any issue 😄

**Comment by Qt-nhannguyen** [comment]
> I don't think it should be our job to proof read these. The letter case differences don't matter that much but wrong country name will surely cause issues in SFDC validation. I didn't go through each single one so there might be something else as well.

I think we should correct the obvious errors. I always try to go through all the added & changed institutions, before pushing the commits. Two eyes on the topic are better than one. Thanks again! 😃

**Review by ollipuu — APPROVED** [review]
Well done, on to production and above! 🚀

**Pull request merged**